src/pages/api/github/pr.ts

```
@@ -49,6 +49,9 @@ async function pullAndUpdatedPR(
     .catch((err) => {
       throw err;
     });
+  if (pullDetails?.data?.merged) {
+    throw "Your transcript has been merged!";
```

I think you might want to throw an error rather than just a plain string